packages/api/lib/rulesHelpers.js

```diff
@@ -558,13 +563,16 @@ async function invokeRerun(rule) {
   await invoke(process.env.invoke, payload);
 }
 
-/*
+/**
  * Updates rule trigger for rule
  *
- * @param {ApiRule}        original- Rule to update trigger for
- * @param {ApiRule}        updates - Updates for rule trigger
- * @param {Knex}           knex    - Knex DB Client
- * @returns {ApiRule}              - Returns new rule object
+ * @param {RuleRecord} original - Rule to update trigger for
+ * @param {Object} updates      - Updates for rule trigger
+ * @param {string} updates.state
+*  @param {Object} updates.rule
+ * @param {unknown} updates.rule.value
+ * @param {Knex} knex           - Knex DB Client
+ * @returns {Promise<RuleRecord>}        - Returns new rule object
  */
 async function updateRuleTrigger(original, updates, knex) {
```

Comment:
This was tricky. One of the main complications in the Rules work is that a lot of methods in the deprecated Rule model were _already_ duplicated here in rulesHelpers, prior to this PR. This PR switches everything over to use the helpers and fully removes the model but there were several problems there. The methods are duplicated but different in subtle ways. These helpers needed to be reconciled with the Rule methods while not breaking existing calls to the helper methods.